### PR TITLE
Frontend for treasury transfer status

### PR DESCRIPTION
**Pull request by Oleh-Zorin**

### CODE REVIEW

File: frontend/src/components/StatusTransfer.tsx [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+                    </div>
+                )
+            })
+        } else {
+            return null
+        }
+    }
+
+
+    useEffect(() => {
```
Comment (tensojka) [inline frontend/src/components/StatusTransfer.tsx:71]: What is the purpose of the refetch() here? It’s not needed elsewhere.

**Review by tensojka — CHANGES_REQUESTED** [review]
I’m not sure why a new PR was opened, but nearly all of the issues have been fixed now, thanks!

Just the one `refetch` left

**Comment by tensojka** [comment]
Supersedes #137

**Comment by tensojka** [comment]
Supersedes #136

File: frontend/src/components/StatusTransfer.tsx (hunk 1 above)
Comment (Oleh-Zorin) [reply frontend/src/components/StatusTransfer.tsx:71]: without refetch get_live_transfers does not make a request

Comment (Oleh-Zorin) [reply frontend/src/components/StatusTransfer.tsx:71]: if you run refetch without useEffect is constant requests and re-render

Comment (tensojka) [reply frontend/src/components/StatusTransfer.tsx]: @Oleh-Zorin I just pushed a version without the refetch. Issue was missing `args: [],` in useContractRead.

My original question was – why do we need refetch *in this component and not elsewhere*. The answer is, args are needed and the refetch provides the (empty) args.

**Review by tensojka — APPROVED** [review]

**Pull request merged**